### PR TITLE
chore: update lint and prettier to qlik config

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+CODEOWNERS
+LICENSE
+coverage
+core
 dist/
-
-*.html
\ No newline at end of file
+node_modules
+yarn.lock
```
Comment (cbt1) [inline .prettierignore:12]: I think you should include the API spec file too. Generated files should never be "prettified" IMO

**Review by cbt1 — APPROVED** [review]
A minor comment only, LGTM

File: .prettierignore (hunk 1 above)
Comment (niekvanstaveren) [reply .prettierignore:12]: Yes!

**Pull request merged**